spec/features/user_login_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+require 'helpers/user_helpers'
+
+feature 'A user tries to login' do
```

Comment:
I don't like this way of structuring tests.  I can see what you want to do, but for me it makes the tests unreadable in code form.  `A user tries to login` is not a feature.